CHANGEDB.php

```diff
@@ -775,5 +775,6 @@
 DELETE FROM `gibbonSetting` WHERE scope='Messenger' AND name IN ('messengerLastBubble','messageBubbleBGColor','messageBubbleWidthType','messageBubbleAutoHide');end
 INSERT INTO gibbonCountry VALUES ('South Sudan', '211');end
 INSERT INTO `gibbonSetting` (`scope`, `name`, `nameDisplay`, `description`, `value`) VALUES ('User Admin', 'publicRegistrationAllowedDomains', 'Public Registration Allowed Domains', 'Comma-separated list of email address domains allowed when registering. Leave blank for no restriction.', '');end
+ALTER TABLE `gibbonActivitySlot` CHANGE `gibbonSpaceID` `gibbonSpaceID` int(10) UNSIGNED ZEROFILL DEFAULT NULL;end
```

Comment:
Hi Ray! I see you merged in the latest changes, thanks. Looks like this line accidentally ended up before the end of the list, which will break the updater. Can you move it to the end? Then it looks like it's ready to merge. Thanks!

Reply:
Hi Sandra. I've moved it to the end and merged in the most recent changes.